mne/inverse_sparse/mxne_inverse.py

```diff
@@ -598,13 +608,30 @@ def tf_mixed_norm(evoked, forward, noise_cov, alpha_space, alpha_time,
     all_ch_names = evoked.ch_names
     info = evoked.info
 
+    if alpha is not None and l1_ratio is not None:
+        old_parametrization = False
+
+        if not (0. <= alpha <= 100.):
```

Comment:
A similar test would be nice for mixed_norm, too. Plus an update of the parameter description, which gives no info for the user how to set the parameter.